rollup.config.js

```diff
@@ -11,17 +11,19 @@ export default {
     external: [
         'react',
         'react-bootstrap',
+        'react-bootstrap/Breadcrumb',
```

Comment:
I wonder why this is required for this, but not e.g. for Button.

Reply:
probably because i import from ```react-boootstrap/Breadcrumb``` and so on according to react-bootstrap docs, but button is only from ```react-boootstrap```